[2024-08-20T10:56:29.576+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-20T10:56:29.598+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_python.create_bucket manual__2024-08-20T10:56:28.490174+00:00 [queued]>
[2024-08-20T10:56:29.612+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_python.create_bucket manual__2024-08-20T10:56:28.490174+00:00 [queued]>
[2024-08-20T10:56:29.613+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-20T10:56:29.635+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): create_bucket> on 2024-08-20 10:56:28.490174+00:00
[2024-08-20T10:56:29.643+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2154) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-20T10:56:29.643+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python', 'create_bucket', 'manual__2024-08-20T10:56:28.490174+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_pyton.py', '--cfg-path', '/tmp/tmptgil7ncm']
[2024-08-20T10:56:29.645+0000] {standard_task_runner.py:72} INFO - Started process 2155 to run task
[2024-08-20T10:56:29.645+0000] {standard_task_runner.py:105} INFO - Job 61: Subtask create_bucket
[2024-08-20T10:56:29.714+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_python.create_bucket manual__2024-08-20T10:56:28.490174+00:00 [running]> on host 93607a8c7584
[2024-08-20T10:56:29.838+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Juancito2085' AIRFLOW_CTX_DAG_ID='dag_with_python' AIRFLOW_CTX_TASK_ID='create_bucket' AIRFLOW_CTX_EXECUTION_DATE='2024-08-20T10:56:28.490174+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-20T10:56:28.490174+00:00'
[2024-08-20T10:56:29.840+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-20T10:56:29.859+0000] {logging_mixin.py:190} INFO - Bucket already exist at E:/Data Science/Tutorial-Apache Airflow/descargas
[2024-08-20T10:56:29.859+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-20T10:56:29.871+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-20T10:56:29.872+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_python, task_id=create_bucket, run_id=manual__2024-08-20T10:56:28.490174+00:00, execution_date=20240820T105628, start_date=20240820T105629, end_date=20240820T105629
[2024-08-20T10:56:29.901+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-20T10:56:29.931+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-20T10:56:29.935+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
